docs/visual/e2e-testing/troubleshooting.md

```diff
@@ -95,3 +95,16 @@ capabilities = {
 ```
 
 Additional information on WebDriver timeouts can be found [here](https://developer.mozilla.org/en-US/docs/Web/WebDriver/Errors/ScriptTimeout).
+
+
+## cropTo can not crop the target element
+
+If you can't crop the target element with `cropTo` and the element isn't scrollable, you can use the `scrollAndStitchScreenshot` parameter. For example:
```

Comment:
```suggestion
When using the [Snapshot command](https://docs.saucelabs.com/visual/e2e-testing/commands-options/#snapshot-command), if you can't crop the target element with `cropTo` and the element isn't scrollable, you can use the `scrollAndStitchScreenshot` parameter. For example:
```
Adds a bit more context in the description of the issue.

Reply:
We should also add the code snippet for the `scrollAndStitchScreenshot` option to this section: https://docs.saucelabs.com/visual/e2e-testing/commands-options/#snapshot-command

Reply:
Thank you for your rewording. Also added the new snippet in the snapshot-command page